OWNERS_ALIASES

```diff
@@ -1,85 +1,148 @@
+# This file is auto-generated from peribolos
+
 aliases:
-  serving-api-approvers:
+  api-core-wg-leads:
```

Comment:
This looks like it might break our existing OWNERS files a bit? Do we need to adjust them first @evankanderson 

/hold

For this.